theme/semantic-tokens.ts

```diff
@@ -4,4 +4,23 @@ import { defineSemanticTokens } from '@pandacss/dev';
 // ts-unused-exports:disable-next-line
 export const semanticTokens = defineSemanticTokens({
   ...leatherSemanticTokens,
+  // TEMPORARY FIX FOR BROKEN COLORS
+  // TODO: Remove when we have all use case semantic token colors
+  colors: {
+    ...leatherSemanticTokens.colors,
+    oldInk: {
```

Comment:
I went through this PR and we seem to only use:
```
oldInk.1
oldInk.2
oldInk.10
oldInk.12
```

So maybe just include these? then we can ask Fab to come up with a semantic name for them? 

On the `containers` branch I found some other ones that seem incorrect. Like in the `highlighter.tsx` file:
```
ink.400
ink.900
```
We also have some colours like that in `test-app/` we can update also

We could audit that file. It seems only used in the errorboundary + contract deploy. I found a typo in there for a space also - ` pr="sapce.04"` 

`home.layout.tsx` uses `ink.1`
`unlock.tsx` uses `ink.2` 
`feedbackbutton.tsx` uses `ink.1` and `ink.2`

Reply:
Some of the `ink` colors with numbers are still from stacks/ui lib so prob just broken atm. Will fix.